docs/dust_extinction/references.rst

```diff
@@ -2,6 +2,43 @@
 References
 ##########
 
+How to Cite
+===========
+
+Please cite the papers presenting the models.  For some models, it is nice to also cite 
+the works that provided a substantial input for that specific model.
+
++--------------+----------------------+
+| Model        | Substantial Input    |
++==============+======================+
+| CCM89        | FM88                 |
++--------------+----------------------+
+| O94          | CCM89                |
++--------------+----------------------+
+| G03          | GC98, MCG99          |
++--------------+----------------------+
+| G23          | GCC09, F19, G21, D22 |
++--------------+----------------------+
+
+In addition, please cite the dust_extinction package 
+(BibTex format found on "Cite this repository" button on main repository page).
```

Comment:
I am not sure if I know where to find this button. Is this on Github or read the docs?

Reply:
Github.  Will clarify.